### PR TITLE
fix 406 not acceptable due to missing headers, remove unit test asser…

### PR DESCRIPTION
**Pull request by danespinosa**
Recently auth started failing due to missing headers in Auth requests, based on https://github.com/dgreif/ring/blob/main/packages/ring-client-api/rest-client.ts added the missing headers.

Updated unit tests, session api doesn't return Auth token anymore. This solves issue #31

### CODE REVIEW

**Comment by chrisstuart-cl** [comment]
This fix works for authentication via 2FA. For refresh token authentication, you also need to do this:
image

**Comment by chrisstuart-cl** [comment]
Tested this update and it works. Many thanks for fixing this.

File: Api/Session.cs [hunk 1]
```diff
@@ -243,12 +247,16 @@ public async Task RefreshSession(string refreshToken)
                 { "refresh_token", refreshToken }
             };
 
+            var headerFields = new NameValueCollection()
+            { 
+                { "hardware_id", "unespecified" }
```
Comment (ronwarner) [inline Api/Session.cs]: Is there a reason "unespecified" is misspelled?  Like, does this match against something Ring cares about, or is it just a nonce string to put something in here for hardware_id and thus the spelling doesn't matter?

**Comment by Audreth** [comment]
implemented the pull requests/code changes and tested as well, they are working. (if you guys need external validation)

**Pull request merged**